Scripts/Macros.cmd.txt

```diff
@@ -0,0 +1,7 @@
+b=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat $*
+br=%VFS_SCRIPTSDIR%\BuildGVFSForWindows.bat Release $*

```

Comment:
Other scripts that I run _all_ the time are `..\BuildOutput\GVFS.Build\InstallG4W.bat` and `..\BuildOutput\GVFS.Build\InstallGVFS.bat`

Reply:
A subsequent PR will condense those to an InstallProduct script (since we require both to be in sync) for the yaml build support. But your use case probably wants them separated still, so you don't need to pay for Git installation when you're iterating on VFS for Git?

Reply:
Yep for my local testing, I only want to pay the cost of installing G4W once, and then I can build/install GVFS over and over again while tweaking things